pkgs/build-support/cc-wrapper/default.nix

```diff
@@ -55,6 +55,26 @@
 , includeFortifyHeaders ? null
 }:
 
+let libcxx_args = libcxx; in
+let
+  stdenv = stdenvNoCC;
+
+  useGccForLibs = useCcForLibs
+    && libcxx_args == null
+    && !stdenv.targetPlatform.isDarwin
+    && !(stdenv.targetPlatform.useLLVM or false)
+    && !(stdenv.targetPlatform.useAndroidPrebuilt or false)
+    && !(stdenv.targetPlatform.isiOS or false)
+    && gccForLibs != null;
+
+  libcxx =
+    if libcxx_args != null
+    then libcxx_args
+    else if useGccForLibs
+    then lib.getLib gccForLibs
+    else null;
```

Comment:
`libcxx` has a connotation with the specific implementation; I think rrbutani's naming was good: `cxxStdlib`